testcafe/tests/photos/photos_crud.js

```diff
@@ -1,10 +1,10 @@
 import { photosUser } from '../helpers/roles' //import roles for login
 import { TESTCAFE_PHOTOS_URL, SLUG } from '../helpers/utils'
-import random from 'lodash/random'
```

Comment:
No link with this PR, but I forgot to remove this line before and prettier was kind enough to remind me to remove it when I edited this file to add a mask